end_app/weather_app/get_weather.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import os
+from dotenv import load_dotenv
+from datetime import datetime
+
+def get_weather_data(city_name):
+    load_dotenv()
+    api_key = os.getenv("api_key")
```

Comment:
a to nie było dodane do settingsów? Jeśli nie to raczej tam powinno być dodane, a później importowane stamtąd